### PR TITLE
Adopt Prettier (code formatting)

### PR DESCRIPTION
**Pull request by lb-**
* Resolves https://github.com/wagtail/guide/issues/114
* Relates to https://github.com/wagtail/guide/issues/151

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
+  "private": true,
+  "author": "",
+  "engines": {
+    "node": "^16"
+  },
+  "scripts": {
+    "start": "webpack --mode=development --watch --progress",
+    "build": "webpack --mode=production",
+    "format": "prettier --write .",
+    "lint:format": "prettier --check .",
```
Comment (thibaudcolas) [inline package.json:12]: Note I think this won’t process dotfiles (`.gitpod.yml`). If we wanted this we’d need a more complex pattern like the one we use in Wagtail.

File: README.md [hunk 2]
```diff
@@ -3,31 +3,32 @@
...
 
-- Python >= 3.9
-- Poetry
-- Git
-- Node 16.*
-- [Yarn](https://yarnpkg.com/)
+-   Python >= 3.9
+-   Poetry
+-   Git
+-   Node 16.\*
```
Comment (thibaudcolas) [inline README.md:30]: Oh interesting

**Review by thibaudcolas — APPROVED** [review]
Looking great! One thing I think we could consider revisit, let me know what you think @lb-.

File: README.md (hunk 2 above)
Comment (lb-) [reply README.md:30]: yeah true, I did not even look at that - not sure why Prettier changed this.

File: package.json (hunk 1 above)
Comment (lb-) [reply package.json:12]: ahh ok - I always wondered why you did that.

Have raised https://github.com/wagtail/guide/issues/162 for an improvement.

**Pull request merged**